bookwyrm/templates/user/groups.html

```diff
@@ -30,6 +35,10 @@ <h1 class="title">
     </div>
 
     {% include 'groups/user_groups.html' with memberships=memberships %}
+
+    {% if not memberships or not memberships.exists %}
```

Comment:
Same as for lists, `membership.exists` is always false. This also should be in `groups/user_groups` rather than here. From line 34 you can do:

```
{% empty %}

<p><em>{% trans "No groups found." %}</em></p>

{% endfor %}
```